dirty_cat/super_vectorizer.py

```diff
@@ -470,22 +505,31 @@ def fit_transform(self, X, y=None):
         ).columns.to_list()
 
         # Classify categorical columns by cardinality
+        _nunique_values = {  # Cache results
+            col: X[col].nunique() for col in categorical_columns
+        }
+        binary_cat_columns = [
+            col for col in categorical_columns if _nunique_values[col] <= 2
```

Comment:
Should this be only if there are binary categorization?
```suggestion
            col for col in categorical_columns if _nunique_values[col] == 2
```
I guess there is no point in encoding if `_nunique_values[col] <= 1`

Reply:
True, but what do you think we should do with the features with cardinality==1 ? I guess maybe drop them ?

Reply:
Yes drop would be the best not to create confusion.

Reply:
So, for now, we decided with Gaël to not drop it for now (given we want to release very soon), but that's something we can discuss in the future.